start-site/src/main/resources/application.yml

```diff
@@ -936,7 +936,7 @@ initializr:
           id: sentry
           bom: sentry
           description: Application performance monitoring and error tracking that help software teams see clearer, solve quicker, and learn continuously.
-          compatibilityRange: "[3.0.0,3.2.0-M1)"
+          compatibilityRange: "[3.0.0,3.3.0-M1)"
```

Comment:
That looks wrong to me.

Reply:
My bad, sorry! I have reverted the change.